pay-api/src/pay_api/schemas/schemas/bank_accounts.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/bank_accounts",
+  "type": "object",
+  "required": [
+  ],
+  "properties": {
+    "accountId": {
```

Comment:
Is this correct? Schema should be having bank number, branch number and account number. Change the keys accordingly.